src/WebJobs.Script/Description/ScriptTypes.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    public class ScriptTypes
```

Comment:
Please move this to Description/DotNet folder to make it clear that this is no longer intended to be a general list of supported languages. You might rename the class DotNetLanguage to make this clearer?

Reply:
Also, I don't think you need the Unknown value?

Reply:
deleted!